### ISSUE TITLE
Screen frozen in Oculus quest headset everything else works fine

### ISSUE THREAD

**Issue by BlaxHazHax** [polygraphene/ALVR#424]
when i start the alvr server No errors appear it says the server is alive,steam vr says my controllers are connected and tracking and i can see my headset tracking in the headset viewer but in the headset the screan is just frozen on the "Press trigger to start steam vr" screen, i am using a 5ghz wifi, my pc meets the necessary requirements, i have tried disabling windows firewall, allowing the ports and alvr, disabling steam sound changing the video codec and lowering the bitrate i am completely lost on what to do :'(
![20190616_194236](https://user-images.githubusercontent.com/51140038/59571358-b5f5cd80-9071-11e9-877d-c006a09c47a3.jpg)

**Comment by Myhamof** [comment]
There is two reasons that i can think of:
1. Maybe the port are blocked by the router try another one or maybe your phone
2. Maybe there was a problem in the installation, maybe try reinstall it

**Comment by polygraphene** [comment]
This kind of issue means you can't receive video frame from server.
Typically, you can get it work by lowering bitrate. Are you getting issue even on 1Mbps?
Or you should check if version of client and server match.

**Comment by BlaxHazHax** [comment]
the issue is still there no matter what bitrate i put it on, i am using the alpha 5 portable for the server and i have the alpha 5 apk sieloaded on my quest

**Comment by InfernalSyndicate** [comment]
Having this same issue. Just installed ALVR 2.4.0-alpha 5 on my Quest with SideQuest and installed the same version 2.4.0-alpha 5 on my PC. Ports 9943-9944 are forwarded and the program is being run as administrator. I've tried reducing it to the lowest streaming bandwidth and it still says 0 packets sent. The Quest shows up in the program and I can connect it and launch the server and SteamVR with my controller. SteamVR appears on my PC with a headset that says H on it but I get no display to my Quest. My headset is connected to 5GHz WiFi and my PC is hardwired to the same router, which shows up with the network name of the 5GHz band. Does my PC need to be on WiFi as well? I've got a PCI WiFi card but I'll have to remove the USB hub I use for my Rift sensors to add it so I'd rather avoid it if possible.

**Comment by Myhamof** [comment]
After some testing i got the same problem when i started alvr from folder containing unicode, try save the alvr folder at drive c:// and start it from there also check that the drivers are installed on the last tab, i suggest you to uninstall and reinstall the driver to be safe

**Comment by BlaxHazHax** [comment]
@Myhamof I tried that with no difference :'(

**Comment by gitHubHenry2** [comment]
I have the exact problem... Don't know if is relevant (newbie here) but I installed by dragging apk into sidequest

**Comment by BlaxHazHax** [comment]
Omg I figured out what it was apparently ALVR has a hard time with some AMD GPU's all I had to do was change the resolution to 75% and it started working

**Comment by gitHubHenry2** [comment]
Some progress... have been trying to get this to work all week lol.  I'm able to a get it running as long as both my connections are wireless. The moment I plug in the Ethernet cable and try to connect will get stuck, says its connected but wont transfer packets. Exactly the same as InfernalSyndicate. I tried opening the ports (running the .bat), turning off my firewalls, trying out different versions of alvr, uninstalling and reinstalling the plugin, nothing. Maybe the problem lies in the router? Somehow dividing the connection between wifi and the ethernet? Don't know what I'm really talking about. I'm running out of ideas... Anybody?

**Comment by InfernalSyndicate** [comment]
I've now tested this on WiFi as well and with a completely different router and it makes no difference. I've reinstalled everything from scratch 3 times now on my PC and from sidequest and ADB commands for the Quest and had to disconnect all of my PC lights to get WiFi back in my computer for literally nothing. I've forwarded ports, added firewall exceptions, run the program in administrator, shit still doesn't work. Is there something I'm missing here? I've meticulously gone through the steps 3 times now and I'm at a loss unless it's just broken.

**Comment by gitHubHenry2** [comment]
Hey! Finally got it working!! I spent so much time working on it Lol. Believe it or not I had two ethernet ports an the back of the PC, both of them working, but I haven't even realized there was two of them (pic attached) so I just switched the connection from 1 to 2. Probably that is not your same issue. I do have another suggestion though (which works too, and actually getting less latency out of it). Try enabling your PC as a hotspot (from your network settings) and connect your Quest to that hotspot. Hopefully that would skip the transfer back to the router and go straight to the Quest. Best of lucks and let us know!
![ports](https://user-images.githubusercontent.com/52017415/60038028-08358080-9668-11e9-8961-c2ea0385c6d9.jpg)
